docs/duration.md

```diff
@@ -516,17 +519,20 @@ quarters = d.months / 3;
 quarters; // => 3
 ```
 
-### duration.**total**(_options_: object) : number
+### duration.**total**(_totalOf_: object) : number
```

Comment:
```suggestion
### duration.**total**(_totalOf_: string | object) : number
```

Reply:
Ugh, thanks for finding these. Fixing now. The positive part about doing all changes via search-n-replace x5 is that all the changes are the same. The negative part is that it's more embarrassing when you forget one! 😄